pegasus/sites.v3/code.org/views/plc_plp_nav.haml

```diff
@@ -1,7 +1,7 @@
 #about_nav
   %span.about_list
-    -current_url = '/educate/plc/plp'
-    -{'Apply to be a Partner'=>'plp-application', 'Terms of Partnership'=>'plp-terms', 'Where we need partners'=> current_url+'#partners', 'FAQ'=>'plp-faq'}.each_pair do |label,url|
+    -current_url = '/educate/professional-learning-partner'
```

Comment:
Is there an easy way to grab this dynamically (e.g. via `request.path_info`?) instead of hardcoding it? Then, we don't have to remember to change this every time we decide to move some pages around.